Mathlib/SetTheory/Descriptive/Tree.lean

```diff
@@ -0,0 +1,31 @@
+/-
+Copyright (c) 2024 Sven Manthe. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sven Manthe
+-/
+import Mathlib.Order.CompleteLattice.SetLike
+
+/-!
+# Trees in the sense of descriptive set theory
+
+This file defines trees of depth ω in the sense of descriptive set theory.
+
+## Main declarations
+
+* `Tree A`: A (possibly infinite) tree of depth at most ω with nodes in A
```

Comment:
English grammar does (usually) lower-cases the first letter after a colon. (This is different from German.) In this instance, capitalising it is particularly confusing.
```suggestion
* `Tree A`: a (possibly infinite) tree of depth at most `ω` with nodes in `A`